$NetBSD$

passdown {C,F}FLAGS and also quote stuff so
CC=gcc -pipe
works

--- Makefile.aimk.orig	Mon Aug 11 10:38:48 1997
+++ Makefile.aimk	Tue May 15 22:29:18 2001
@@ -34,9 +34,9 @@
 #
 
-PVMOPTS		=	CC=$(CC) \
-				F77=$(F77)
+PVMOPTS		=	CC="$(CC)" \
+			F77="$(F77)" \
+			CFLOPTS="$(CFLAGS)" \
+			FFLOPTS="$(FFLAGS)" 
 
-#				CFLOPTS=$(CFLOPTS)
-#				FFLOPTS=$(FFLOPTS)
 #				PVMDIR=$(PVMDIR)
 
